tests/lib/rules/func-style.js

```diff
@@ -185,6 +200,17 @@ ruleTester.run("func-style", rule, {
                 }
             ]
         },
+        {
+            code: "var foo = () => ({ bar() { super.baz(); } });",
```

Comment:
Is this different from the function declaration version? I.e.:

```js
function foo () {
    return { bar() { super.baz(); } };
}
```

Reply:
I believe it isn't different, that's why the rule reports it (this is an `invalid` test case).